packages/arb-token-bridge-ui/src/hooks/useArbQueryParams.tsx

```diff
@@ -177,6 +175,44 @@ export const ChainParam = {
   decode: decodeChainQueryParam
 }
 
+export function encodeTabQueryParam(
+  tab: string | (string | null)[] | null | undefined
+): string {
+  if (!tab || Array.isArray(tab)) {
```

Comment:
`tab` type doesn't match the definition here. We get number but expect a string.

This also means that we are checking for `!0` and default to Bridge for every first tab. Bridge won't always be the first tab though and so later we're going to have a bug here

Reply:
What about something like this:
```
enum Tab {
    BRIDGE = 'bridge',
    TX_HISTORY = 'tx_history'
}
const tabToIndex = {
    [Tab.BRIDGE]: 0,
    [Tab.TX_HISTORY]: 1,
} as const satisfies Record<Tab, number> 
const indexToTab = {
    0: Tab.BRIDGE,
    1: Tab.TX_HISTORY
} as const satisfies Record<number, Tab>
```

So we don't need to use `!` and better typing